src/Faker/Provider/fr_FR/Address.php

```diff
@@ -44,13 +44,26 @@ class Address extends \Faker\Provider\Address
     );
 
     private static $regions = array(
+        // Les treize régions métropolitaines, par ordre alphabétique de leurs noms et codes ISO 3166-2 :
```

Comment:
Is there are source for this?

Reply:
New region in france in 2017
https://fr.wikipedia.org/wiki/R%C3%A9gion_fran%C3%A7aise